cypress/integration/code-schools.spec.js

```diff
@@ -33,7 +33,16 @@ describe('code schools', function() {
     });
   });
 
-  // /Online Schools test
+  // Online Schools test
+  it('renders only online code schools after clicking on "Online Schools"', () => {
```

Comment:
just a nit with the test name: `renders strictly online-only code schools...`

Reply:
Would "only renders online code schools..." remove confusion better, since it is all schools with an online option and not online-only schools?

Reply:
Oh shoot! My mistake.

How about... `only renders code schools with online education`

Reply:
Had some issues with the linter and the line being too long. How about: `only renders online schools with an online option after clicking "Online Schools"`